core/src/contracts/update.rs

```diff
@@ -9,7 +9,7 @@ use thiserror::Error;
 
 #[derive(Clone)]
 pub struct UpdateContract {
-    ergo_tree: ErgoTree,
+    pub ergo_tree: ErgoTree,
```

Comment:
Could you please add a getter for `ergo_tree` property like in other contracts? I'm still not sure if we'll keep `ErgoTree` or generate it on request, so it's better to keep it private.